ocaml/tests/test_repository_helpers.ml

```diff
@@ -952,8 +874,9 @@ module EvalGuidanceForOneUpdate = Generic.MakeStateless (struct
                       id= "UPDATE-0001"
                     ; summary= "summary"
                     ; description= "description"
-                    ; rec_guidance= Some Guidance.RestartDeviceModel
+                    ; rec_guidance= Some RestartDeviceModel
                     ; abs_guidance= None
+                    ; guidance= [(Mandatory, [RestartDeviceModel])]
                     ; guidance_applicabilities=
                         [
                           Applicability.
```

Comment:
Some comments seems need update, like
line 888: "old version 0.2.1 is greater than 0.2.0"

Reply:
The "old version" means the current installed version in an update. Here it means the applicability is not applicable to the update.